androidHyperskillApp/src/main/res/values/strings.xml

```diff
@@ -18,6 +18,13 @@
     <!-- Notification channel names -->
     <string name="other_channel_name">Other</string>
     <string name="daily_reminder_channel_name">Daily reminders</string>
+    <string name="daily_reminder_channel_description">Daily reminders about studying and staying motivated</string>
+    <string name="streak_boosters_channel_name">Streak boosters</string>
+    <string name="streak_boosters_channel_description">Learning streak maintenance notifications</string>
+    <string name="streak_savers_channel_name">Streak savers (freezers)</string>
+    <string name="streak_savers_channel_description">Learning streak freeze notifications</string>
+    <string name="regular_learning_reminders_channel_name">Regular learning reminders</string>
+    <string name="regular_learning_reminders_channel_description">Regular study reminders less often than every day</string>
```

Comment:
I'll ask the marketing team is this texts OK, maybe they will propose some changes

Reply:
https://vyahhi.myjetbrains.com/youtrack/issue/ALTAPPS-835/Android-Setup-push-notification-showing#focus=Comments-74-239478.0-0